### PR TITLE
Another Shutdown Banner

### PR DESCRIPTION
**Pull request by skalnik**
This builds on https://github.com/CAVaccineInventory/vaccinatethestates/pull/337 by @MoralCode. I mostly removed a bunch of markup and used the tailwind css classes to make it look pretty similar.

Additionally I removed the `Volunteer` link as prompted in Discord!

Link to Deploy Preview: https://deploy-preview-338--vaccinatethestates.netlify.app/

---

### Manual Testing (QA)

_(Note: Use your best judgement for time management. If this PR is a minor content adjustment, you might not invest in the full list of QA checks below. But for medium-large PRs, we recommend a thorough review.)_

- [x] I solemnly swear that I QA'd my change. My change does not break the site on `localhost` nor staging.

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ home_title: Vaccination Sites Near Me
 aboutus_title: About Us
 vaccinebot_title: VaccineBot
 volunteer_title: Volunteer
+shutdown: We have shut down, all data stopped being updated on YYYY-MM-DD
```
Comment (skalnik) [inline _i18n/en.yml:5]: Need an actual date for this!

**Comment by eliblock** [comment]
This looks good to me on the main page!

**Review by eliblock — COMMENTED** [review]
This looks good to me on the main page!

File: index.html [hunk 2]
```diff
@@ -7,6 +7,18 @@
 <link href="assets/css/near-me.css" rel="stylesheet" />
 <div class="flex flex-col items-center">
   <div class="container mx-auto mt-4 lg:mt-8">
+    <div class="border-2 border-red-600 rounded-md flex bg-red-100 place-items-center
```
[the remark this answers is not in the document]
Comment (skalnik) [reply index.html:10]: About us makes sense, but I think we can hold off on the embed as we only just notified folks. What do you think?

[the remark this answers is not in the document]
Comment (eliblock) [reply index.html:10]: Makes sense to me!

**Review by eliblock — APPROVED** [review]

**Pull request merged**